### PR TITLE
So 5213

### PR DESCRIPTION
**Pull request by LeonardoHabitzreuter**
Ticket:
https://teamsnap.atlassian.net/browse/SO-5213

The purpose of this PR is to improve the feedback for users when PaginatadTable has no data to show

### CODE REVIEW

File: src/js/components/Table/Paginated/PaginatedTable.tsx [hunk 1]
```diff
@@ -173,7 +175,7 @@ const PaginatedTable: PaginatedTableProps = ({
       setIsLoading(false);
       if (data) setDataSet(data);
 
-      if (currentTotalItems) setTotalItems(currentTotalItems);
+      if (typeof currentTotalItems === 'number') setTotalItems(currentTotalItems);
```
Comment (bradcypert) [inline src/js/components/Table/Paginated/PaginatedTable.tsx:178]: Why is this change needed?

Comment (LeonardoHabitzreuter) [reply src/js/components/Table/Paginated/PaginatedTable.tsx:178]: We need to update the totalItems state even when the "currentTotalItems" is 0, before this change the if condition was returning false

Comment (LeonardoHabitzreuter) [reply src/js/components/Table/Paginated/PaginatedTable.tsx:178]: I thought on code setTotalItems(currentTotalItems || 0) but I don't know if this can be some side effects

File: src/js/components/Table/Paginated/PaginatedTable.tsx [hunk 2]
```diff
@@ -63,6 +63,7 @@ const propTypes = {
   rowsAreSelectable: PropTypes.bool,
   searchPlaceholder: PropTypes.string,
   defaultSort: PropTypes.string,
+  placeholder: PropTypes.string,
```
Comment (jondauz) [inline src/js/components/Table/Paginated/PaginatedTable.tsx:66]: I would suggest to change the param name to be something more descriptive than `placeholder` such as `noResultsText`.

**Review by jondauz — CHANGES_REQUESTED** [review]

**Review by bradcypert — APPROVED** [review]
